docs/openshift-on-azure-walkthrough.md

```diff
@@ -195,11 +199,125 @@ oc login <FQDN> --token=<YOUR_TOKEN>
 ```
 
 
+## AAD application configuration
```

Comment:
this needs to go further towards the beginning of the doc so that folks know it's a prereq to work out the option they're going to use.  Can link to creation steps further in doc if you want to keep the prereq section short.

Reply:
Add small pre-reqs sentence to link to this? Not sure if we can move it all to pre-reqs

Reply:
yeah, we can link down to the content.

Reply:
@mjudeikis the AAD application steps already mentioned in the documentation is all we need for an OSA cluster create scenario. We would like the steps mentioned on this page to be focused around using CLI. If needed another page can be added for creating AAD app in other ways.